examples/fastify/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "dependencies": {
+    "@arcjet/fastify": "1.0.0-beta.9",
+    "@arcjet/inspect": "1.0.0-beta.9",
+    "@fastify/autoload": "6.3.1",
+    "@fastify/env": "5.0.2",
+    "@fastify/type-provider-json-schema-to-ts": "5.0.0",
```

Comment:
I chose this over the type-box version used in the demo simply because it was listed first in the fastify docs